test/server_test.js

```diff
@@ -106,6 +110,33 @@ describe('server', function() {
     });
   });
 
+  it('converts ANSI codes to HTML from the error stack', function() {
+    let error = new Error('whoops');
+    error.stack = `\u001b[35m102\u001b[39m\u001b[33m\u001b[0m`;
+
+    const mockUI = new MockUI();
+    const builder = new Builder(new plugins.Merge([new plugins.Failing(error)]));
+    const watcher = new Watcher(builder, []);
+
+    server = new Server.Server(watcher, '127.0.0.1', PORT, undefined, mockUI);
+    server.start();
+
+    return new Promise((resolve, reject) => {
+      server.http.on('listening', resolve);
+      server.http.on('close', reject);
+      server.http.on('error', reject);
+    }).then(() => {
+      return got(`http://127.0.0.1:${PORT}/`).then(
+        () => {
+          throw new Error('should not be reached');
+        },
+        err => {
+          expect(err.body).to.include(`<span style="color:#ff00ff;">102</span>`);
```

Comment:
@stefanpenner, please check if this is fine. The current version of got in this repo doesn't support getting the body from errors. I had to update it to v9 to get it working. Hope that's fine.

Reply:
Looks like an excellent test to me.